data/devices/coolermaster-cmstorm-recon.device

```diff
@@ -0,0 +1,9 @@
+[Device]
+Name=CoolerMaster CMStorm Recon
```

Comment:
Should be "CoolerMaster CM Storm Recon". There should be a space between CM and Storm.